classes/PodsMeta.php

```diff
@@ -1021,10 +1023,38 @@ public function meta_post ( $post, $metabox ) {
                         if ( isset( $field[ 'help' ] ) )
                             unset( $field[ 'help' ] );
                     ?>
-			<div class="pods-submittable-fields">
-                    <?php echo PodsForm::field( 'pods_meta_' . $field[ 'name' ], $value, $field[ 'type' ], $field, $pod, $id ); ?>
-                    <?php echo PodsForm::comment( 'pods_meta_' . $field[ 'name' ], $field[ 'description' ], $field ); ?>
-			</div>
+			        <div class="pods-submittable-fields">
+                        <?php echo PodsForm::field( 'pods_meta_' . $field[ 'name' ], $value, $field[ 'type' ], $field, $pod, $id ); ?>
+                        <?php echo PodsForm::comment( 'pods_meta_' . $field[ 'name' ], $field[ 'description' ], $field ); ?>
+                    </div>
+                    <?php
+                    if ( 'pick' == $field[ 'type' ] && 'post_type' == $field[ 'pick_object' ] ) {
+                        $url = add_query_arg(
+                            array(
+                                'post_type'  => $field[ 'pick_val' ],
+                                'pods_modal' => '1',
+                                'TB_iframe'  => 'true',
+                                'width'      => '753',
+                                'height'     => '798',
+                            ),
+                            admin_url( 'post-new.php' )
+                        );
+
+                        $pick_format = 'pick_format_' . $field[ 'options' ]['pick_format_type' ];
+                        $field_type = $field[ 'options' ][ $pick_format ];
+                        $pods_relationship_popup_data[ $field[ 'name' ] ] = array(
+                            'url'             => $url,
+                            'field_type'      => $field_type,
+                            'options'         => $field,
+                            'id'              => $id,
+                            'name'            => 'pods_meta_' . $field[ 'name' ],
+                            'value'           => $value
+                        );
+                        // Not ideal, this will result in multiple definitions for multiple pick fields, but the last one wins anyway.
+                        wp_localize_script( 'pods', 'pods_relationship_popup_data', $pods_relationship_popup_data );
+                        ?>
+                        <a href="#" id="pods-related-edit-<?= $field[ 'name' ]; ?>" class="button pods-related-edit" data-field-name="<?= $field[ 'name' ]; ?>" style="margin-top: 1em;">Add New</a>
```

Comment:
The day will come when 5.4 or better is minimum and I can finally stomp on this argument!